amrit-local-setup/README.md

```
@@ -1,100 +1,642 @@
+Auth Database: admin
+```
+
+**Redis:**
+```
+Host: localhost
+Port: 6379
+Password: (none - no authentication by default)
+```
+
+**Elasticsearch:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)